### PR TITLE
enh(Listing): Display skeleton when table is loading at first time

### PR DESCRIPTION
**Pull request by Thebarda**

### CODE REVIEW

File: src/Listing/Skeleton.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  BodyTableCell;
+}
+
+const ListingLoadingSkeleton = ({
+  columnConfiguration,
+  checkable,
+  BodyTableCell,
+}: LoadingSkeletonProps): JSX.Element => (
+  <>
+    {new Array(10).fill(0).map(() => (
```
Comment (kduret) [inline src/Listing/Skeleton.tsx]: 10 could be a prop cause we may know the number of items per page we expect

File: src/Listing/index.stories.tsx [hunk 2]
```diff
@@ -79,3 +79,11 @@ const Story = (props): JSX.Element => {
 };
 
 export const normal = (): JSX.Element => <Story />;
+
+export const loadingWithNoData = (): JSX.Element => {
+  return <Story tableData={[]} totalRows={0} loading />;
+};
+
+export const loadingWithData = (): JSX.Element => {
+  return <Story loading />;
```
Comment (kduret) [inline src/Listing/index.stories.tsx]: using wave animation does not break snapshot testing ?

Comment (bdauria) [reply src/Listing/index.stories.tsx]: We have an error threshold that should cover this case

Comment (Thebarda) [reply src/Listing/index.stories.tsx]: I have

**Pull request merged**